server/protocols/jmap/src/main/java/org/apache/james/jmap/model/VacationResponse.java

```diff
@@ -38,73 +39,67 @@
 @JsonDeserialize(builder = VacationResponse.Builder.class)
 public class VacationResponse {
```

Comment:
It seems this VacationResponse object is used for many purpose:
- build the response
- handle the creation query
- handle the update query
Usually we create different objects for all these purpose, but as you use api objects too, I'm not sure it is necessary.
So I ask for other team members advice :)